problems/getCountriesSortedByPopulation.js

```diff
@@ -7,8 +7,31 @@
 * 
 */
 
-function getCountriesSortedByPopulation() {
+let ex1 = [
+    {
+      country: "Algeria",
+      population: 41,
+    },
+    {
+      country: "Belize",
+      population: 0.4,
+    },
+    {
+      country: "China",
+      population: 1386,
+    },
+    {
+      country: "Denmark",
+      population: 6,
+    },
+  ]
 
+function getCountriesSortedByPopulation(arr) {
+    let sortedArr = arr.sort((a, b) => b.population - a.population)
```

Comment:
Once the array is sorted which array method can we chain that will return a new array? Consider how to key into key/value pairs in the object for your conditional.